### ISSUE TITLE
m3u-filter process crashes after a while

### ISSUE THREAD

**Issue by Robot1982** [euzu/m3u-filter#53]
I start the process with:

./m3u-filter -s

It works for a while (a few hours or less) and then the process crashes with:

`thread 'main' panicked at src/model/xmltv.rs:72:72:`
`called 'Option::unwrap()' on a 'None' value`
`note: run with 'RUST_BACKTRACE=1' environment variable to display a backtrace`

I run Debian 10.13 and m3u-filter version 1.1.8.

**Comment by euzu** [comment]
@Robot1982  i have pushed a new version to master. Please give me a feedback after testing.

**Comment by euzu** [comment]
@Robot1982 could you test it, if not i will close this isue as solved.

**Comment by Robot1982** [comment]
@euzu Yes, it seems that the issue is solved. It runs since 3 days without a crash. I couldn't test it for a longer time because I had problems compiling it. It always failed with a timeout. I had to modify the Dockerfile to make it work. I added "--network-timeout 600000" like this:

`RUN yarn install \
  --prefer-offline \
  --frozen-lockfile \
  --non-interactive \
  --production=false \
  --network-timeout 600000
`

I don't know if this problems only applies to me and wether 600000 is a good value, however this worked for me.
